### PR TITLE
Estimate project size

### LINKED ISSUE #26: Estimate project size
There is no estimate in the project yet. We have to create one as explained [here](https://github.com/teamed/est)

### PR DESCRIPTION
**Pull request by Suseika**
Fixes #26

### CODE REVIEW

**Comment by Suseika** [comment]
@dmarkov Would you please review this pull request?

**Comment by dmarkov** [comment]
@suseika Thanks, I will find someone to review this PR soon

**Comment by dmarkov** [comment]
@longtimeago please review

File: est/jcabi-s3.est [hunk 1]
```diff
@@ -0,0 +1,13 @@
+date: 02-06-2015
+author: Georgy Vlasov
+method: champions.pert
+scope:
+    1: implementations of Buckets, Ockets and Regions
+    2: project site
+    3: achieve 80% test coverage
+    4: CI
+champions:
+    1:
```
Comment (longtimeago) [inline est/jcabi-s3.est]: @Suseika 1 champion is not enough. We need at least 2.
Please, see the README https://github.com/teamed/est/blob/master/README.md

Comment (Suseika) [inline est/jcabi-s3.est]: @longtimeago My bad. Added the second champion.

**Comment by longtimeago** [comment]
@Suseika thank you!

**Comment by longtimeago** [comment]
@rultor merge

**Comment by dmarkov** [comment]
> @dmarkov Would you please review this pull request?

@suseika thanks, will take a look

**Comment by yegor256** [comment]
@rultor merge

**Comment by dmarkov** [comment]
@longtimeago Thanks so much! Your account was topped up for **17 mins** (transaction ID is `58699883`, the task took 24 hours and 29 mins)

extra minutes for review comments (c=2)

+17 added to your rating, at the moment it is: [+3837](http://www.netbout.com/b/35024?open=rating)

**Comment by dmarkov** [comment]
@rultor deploy

**Pull request merged**